Add upstream dependencies guide to docs (#12351)
PrefectHQ/prefect: docs/guides/determining-upstream-dependencies.md

---
description: Run tasks in a desired order
tags:
  - tasks
  - dependencies
  - wait_for
search:
  boost: 2
---

# Determining Upstream Dependencies

> Results from a task can be provided to other tasks (or subflows) to determine their upstream dependencies. Prefect uses upstream dependencies in two ways:

Reviewer: ```suggestion
Results from a task can be provided to other tasks (or subflows) as upstream dependencies. Prefect uses upstream dependencies in two ways:
```
